examples/bri-3/src/bri/identity/bpiSubjects/models/bpiSubject.ts

```diff
@@ -1,18 +1,22 @@
-import { IBpiSubject } from "./bpiSubject.interface";
 import { BpiSubjectType } from "./bpiSubjectType.enum";
 
-export class BpiSubject implements IBpiSubject {
+export class BpiSubject {
     // Fields of the domain model are always changed through it's methods
-    id: string; // TODO: Add uuid after #491
-    name: string;
-    description: string;
-    type: BpiSubjectType;
-    publicKey: string;
+    private id: string; // TODO: Add uuid after #491
+    private name: string;
+    private description: string;
+    private type: BpiSubjectType;
+    private publicKey: string;
 
-    constructor(name: string, description: string, type: BpiSubjectType, publicKey: string) {
-        this.name = name;
-        this.description = description;
-        this.type = type;
-        this.publicKey = publicKey;
+    constructor(
```

Comment:
Please add id to the ctor of every entity we have so far, for consistency sake. We still need to decide who will be responsible for generating new IDs in case of CREATE, but the decision can be postponed for later.

Reply:
done